config/initializers/5_telemetry_ruby.rb

```diff
@@ -0,0 +1,17 @@
+telemetry = Foreman::Telemetry.instance
+
+GC_METRICS = {
+  :count => :ruby_gc_count,
+  :major_gc_count => :ruby_gc_major_count,
+  :minor_gc_count => :ruby_gc_minor_count
+}
+begin
+  GC.stat :total_allocated_objects
+rescue ArgumentError
+  GC_METRICS.update :total_allocated_object => :ruby_gc_allocated_objects, :total_freed_object => :ruby_gc_freed_objects
+else
+  GC_METRICS.update :total_allocated_objects => :ruby_gc_allocated_objects, :total_freed_objects => :ruby_gc_freed_objects
+end
+GC_METRICS.each do |ruby_key, metric_name|
+  telemetry.add_counter(metric_name, "Ruby GC statistics per request (#{ruby_key})", [:controller, :action])
+end
```

Comment:
Could the code in this file live under `lib/foreman/telemetry` ? If this is required to be initialized here, could the code live there but be a one line call out to it? I think it might be more obvious code wise to have everything under `lib/foreman/telemetry` witha simple call here like `telemetry.configure_ruby`